### PR TITLE
RDoc

### PR DESCRIPTION
**Pull request by alexsnaps**
Added some (what I think may well be valid) documentation to this thing

### CODE REVIEW

File: ext/gctrack/gctrack.c [hunk 1]
```diff
@@ -100,6 +121,26 @@ gctracker_start_record(int argc, VALUE *argv, VALUE klass)
...
 }
 
+/*
+ * call-seq:
+ *    GC::Tracker.end_record  -> Array or nil
+ *
+ * Ends the recording of the last started record and return the 
+ * the collected information. 
+ * The array contains two numbers: The amount of GC cycles observed and 
+ * the cumulative duration of these cycles. If multiple recordings were started
```
Comment (maximebedard) [inline ext/gctrack/gctrack.c]: Worth nothing that the duration is captured in nanosec?

Comment (alexsnaps) [inline ext/gctrack/gctrack.c]: s/duration/duration in ns/

Comment (csfrancis) [inline ext/gctrack/gctrack.c]: May want to specify the units for the duration.

File: ext/gctrack/gctrack.c [hunk 2]
```diff
@@ -118,12 +159,35 @@ gctracker_end_record(int argc, VALUE *argv, VALUE klass)
...
 gctracker_enabled_p(int argc, VALUE *argv, VALUE klass)
 {
   return gctracker_enabled() ? Qtrue : Qfalse;
 }
 
+/*
+ * call-seq:
+ *    GC::Tracker.enable   -> true or false
+ *
+ * Returns +true+ if the Tracker has been enabled, +false+ otherwise.
```
Comment (maximebedard) [inline ext/gctrack/gctrack.c]: I believe the description here needs to be swapped with the previous function.

**Review by csfrancis — APPROVED** [review]

**Comment by alexsnaps** [comment]
Added the [README.md](https://github.com/Shopify/gctrack/blob/75248574202d165add0d05d55b821ad3d9d45b82/README.md)

**Comment by alexsnaps** [comment]
@maximebedard I addressed your concerns I think... and to you & @csfrancis I've added a somewhat more complete `README.md`, anything you'd want to see added?

**Review by maximebedard — APPROVED** [review]

**Pull request merged**